packages/core/src/types.ts

```diff
@@ -209,6 +211,7 @@ export const OutputMode = {
   SPLIT: 'split',
   TAGS: 'tags',
   TAGS_SPLIT: 'tags-split',
+  OPERATION: 'operation',
```

Comment:
Please allow me to ask a question to include my understanding
Why do we need an operational mode?
I think other modes such as `split` will work usefully in `hono` as well. If manual updating is necessary, I thought it would be sufficient to clearly state in the guide a rule that only the `builder` of `hono` should not update the file. Or it may be a good idea to specify it with another option such as override. I thought we shouldn't add new modes if we can do it without reducing complexity.

What do you think?

Reply:
The operation mode is not even used by the hono client. I did the development of it at the beginning. It didn't work how I wanted for it. I thought it was a good feature to keep too. To be able on every client to have a operation generation by file. But if you think it's not useful I can remove it or move it to another pull request

Reply:
> The operation mode is not even used by the hono client. 

thank you for teaching me. 
Currently, other clients output corrupted files and do not work, so since this mode is common to all clients, I would suggest moving to another PR.

Reply:
Just removed the code

Reply:
thank you. I have confirmed the changes !